content/reroils.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "RERO ILS: système de gestion de bibliothèque Open Source"
+language: "fr"
+date: 2020-07-09
+---
+
+{{< smallcaps "rero ils" >}} est un système de gestion de bibliothèque (SGB) Open Source développé par RERO en collaboration avec l'[Université catholique de Louvain (UCLouvain)](https://uclouvain.be/). ILS est une abréviation anglaise signifiant *Integrated Library System*. Il s'agit d'une catégorie de logiciels permettant de gérer les activités courantes d'une bibliothèques (acquisitions de documents, prêt, catalogage, recherche) et offrant une interface publique pour les utilisateurs.
+
+D'un point de vue technique, {{< smallcaps "rero ils" >}} se fonde sur le *framework* [Invenio](https://invenio-software.org) développé par le [CERN](https://home.cern/).
+
+Un site web de démonstration est publiquement accessible à l'adresse suivante : [ils.test.rero.ch](http://ils.test.rero.ch "Site de démonstration de RERO ILS").
+
+### Le contexte RERO
+
+{{< smallcaps "rero ils" >}} représente la pierre angulaire du projet de mutation [RERO 21](/about).
+
+Se basant sur son expérience de plus de 30 de gestion d'un important réseau de bibliothèques, la centrale RERO a débuté le développement de {{< smallcaps "rero ils" >}} en 2017, dans le but de remplacer à l'horizon 2021 le logiciel propriétaire actuellement implémenté dans le réseau. Grâce à un système Open Source, la centrale peut exploiter le plein potentiel de ses compétences pour satisfaire au mieux les besoins actuels et futurs de ses bibliothèques membres.
```

Comment:
please add the word "ans" next to 30 in the sentence 
Se basant sur son expérience de plus de 30 <ans> de gestion d'un important réseau de bibliothèques